modxcms/revolution
# [2.x] Fix inability to drag/drop files and folders in tree

### What does it do?
Removes improper application of url and entity encoding on File tree config's id parameters.

### Why is it needed?
To re-enable the drag/drop of files and folders in the File tree, while also preserving the ability to use multibyte characters in folder and file names.

### How to test
Create a handful of folders and files in your tree, with some having mb characters such as ö, and verify that all can be interacted with as expected (moved, renamed, etc.).

### Related issue(s)/PR(s)
Resolves #16194, reverts PR #16071
